homeassistant/components/generic/strings.json

```diff
@@ -38,8 +38,12 @@
           "content_type": "Content Type"
         }
       },
-      "confirm": {
-        "description": "[%key:common::config_flow::description::confirm_setup%]"
+      "user_confirm_still": {
+        "title": "Preview",
+        "description": "![Camera Still Image Preview]({preview_url})",
```

Comment:
Why can't we put the whole thing in the placeholder ?

Reply:
The first part is the text to translate (I assume it is the hover-over/accessibility text for the image)

Reply:
I have coped the pattern from here:
https://github.com/home-assistant/core/blob/fe68c15a4a4a703081c9b9b810b1f09608c6714c/homeassistant/components/hue/strings.json#L18

Having said that, I can't seem to access any accessibility text, so I'm not sure of the purpose of "Camera Still Image Preview" text in strings.json (doesn't seem to be used anywhere)